aspnet/DropDownList/Validation.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Validation in DropDownList control for Syncfusion ASP.NET WebForm
-description: Describes about the Validation in DropDownList control for Syncfusion ASP.NET WebForm
+title: Validation in  ASP.NET WebForm Dropdown List control | Syncfusion
```

Comment:
Update as Validation in ASP.NET Web Forms Dropdown List Control | Syncfusion

Reply:
resolved